### ISSUE TITLE
Improve web.dev accessibility score

### ISSUE THREAD

**Issue by guilhermejordan** [janraasch/hugo-bearblog#6]
Hello! Thanks for the awesome theme.

[I tested the demo site using web.dev](https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fthemes.gohugo.io%2Ftheme%2Fhugo-bearblog%2F), which showed a result of 75 out of 100 in Accessibility.

What seems to be affecting the results are the lines

https://github.com/janraasch/hugo-bearblog/blob/6e2f054d42ecb59b37afcf2069f0a3af550e6638/layouts/_default/baseof.html#L7

and

https://github.com/janraasch/hugo-bearblog/blob/6e2f054d42ecb59b37afcf2069f0a3af550e6638/layouts/partials/head.html#L3

I'm a beginner in this whole HTML thing. How would [the proposed solution](https://web.dev/meta-viewport/?utm_source=lighthouse&utm_medium=lr#how-to-avoid-disabling-browser-zoom) affect the websites based on this theme?

**Comment by janraasch** [comment]
Hi @guilhermejordan, thank you for bringing this up 👍🏻.

To be honest I did not think about this at all. I simply copied the code from the original »Bear Blog« source code, see https://github.com/HermanMartinus/bearblog/blob/master/templates/base.html#L7.

It seems like iOS ignores the declarations of `user-scalable`, `minimum-scale`, and `maximum-scale`, as of iOS 10, see https://sitebulb.com/hints/mobile-friendly/the-viewport-meta-tag-prevents-the-user-from-scaling/. 

Hence we might as well remove this.

Now everybody's happy 💯.

I hope that helps 😸.

Kind regards,
Jan
